meta.yml

```diff
@@ -10,12 +10,19 @@ description: "How federal government domains are meeting best practices on the w
 # Data sources. These URLs should have their content updated in-place.
 data:
   # URL where we download the .gov domain list from.
-  domains_url: "https://raw.githubusercontent.com/GSA/data/gh-pages/dotgov-domains/current-federal.csv"
+  domains_url: "https://github.com/GSA/data/raw/master/dotgov-domains/current-federal.csv"
```

Comment:
The DotGov program has moved their URLs away from `gh-pages` and to `master`. See https://github.com/GSA/data/issues/86